balanceRobot.ino

```diff
@@ -90,4 +78,23 @@ void loop(void) {
   } 
 }
 
-void read3D
+void read3D {
```

Comment:
Naming is not clear, please change the function name to `readAccelerometerData()` or something like that.